Source/UserSession/ZMUserSession+SecurityClassification.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ZMUserSession+SecurityClassification.swift
+//  WireSyncEngine-ios
+//
+//  Created by Sun Bin Kim on 02.02.22.
+//  Copyright © 2022 Zeta Project Gmbh. All rights reserved.
+//
+
+import Foundation
+
+public enum SecurityClassification {
+    case none
+    case classified
+    case notClassified
+}
+
+extension ZMUserSession {
+
+    public func classification(with users: [UserType]) -> SecurityClassification {
+        guard isSelfClassified else { return .none }
+
+        var isClassified = true
+        for user in users {
+            let classification = classification(with: user)
+
+            if classification != .classified {
+                isClassified = false
+                break
+            }
+        }
```

Comment:
I believe you can simplify this with:

```suggestion
        let isClassified = users.allSatisfy {
            classification(with: $0) == .classified
        }
```